static/app/stores/guideStore.tsx

```diff
@@ -73,19 +81,19 @@ const defaultState: GuideStoreState = {
   prevGuide: null,
 };
 
-interface GuideStoreDefinition extends StoreDefinition {
+interface GuideStoreInterface extends StoreDefinition, CommonStoreDefinition<State> {
```

Comment:
Why do we name it interface when the type is already declared as an interface?
`interface NameInterface` sounds redundant and is also less accurate as GuideStoreDefinition as we do not create a class when we define store definition on L96 - it only becomes a store interface inside reflux's createStore (see createStore signature which accepts a StoreDefinition, not a StoreInterface). This was the reason why I change this in the first place because I wanted our type definitions to more accurately reflect what is going on.

Reply:
no idea why i changed this lol